quickstart/101-attestation-provider/main.tf

```diff
@@ -1,22 +1,10 @@
-terraform {
-
-  required_version = ">=0.12"
-    
-  required_providers {
-    azurerm = {
-      source = "hashicorp/azurerm"
-      version = "~>2.0"
-    }
-  }
-}
-
-provider "azurerm" {
-  features {}
+resource "random_pet" "rg-name" {
+  prefix    = var.resource_group_name_prefix
```

Comment:
Can we run `terraform fmt` command for this directory please? The formatted line should be:

```hcl
prefix = var.resource_group_name_prefix
```